main.js

```diff
@@ -7,11 +7,40 @@ document.addEventListener('keydown', turnCar);
 function turnCar(event) {
   if (event.keyCode === 37) /* left */ {
     $racecar.className = 'west';
+    data.orientation = 'west';
   } else if (event.keyCode === 38) /* up */ {
     $racecar.className = 'north';
+    data.orientation = 'north';
   } else if (event.keyCode === 39) /* right */ {
     $racecar.className = 'east';
+    data.orientation = 'east';
   } else if (event.keyCode === 40) /* down */ {
     $racecar.className = 'south';
+    data.orientation = 'south';
   }
 }
+
+var x = 0;
+var y = 0;
+
+function startCar() {
+  if (data.orientation === 'east') {
+    x += 5;
+    $racecar.style.left = x + 'px';
+    data.x = x;
+  } else if (data.orientation === 'west') {
+    x -= 5;
+    $racecar.style.left = x + 'px';
+    data.x = x;
+  } else if (data.orientation === 'south') {
+    y += 5;
+    $racecar.style.top = y + 'px';
+    data.y = y;
+  } else if (data.orientation === 'north') {
+    y -= 5;
+    $racecar.style.top = y + 'px';
+    data.y = y;
+  }
+}
+
+setInterval(startCar, 16);
```

Comment:
You shouldn't start it immediately, the user should press space to start the car

Reply:
Gotcha, fixed it! 
![user-can-start-their-car 02](https://user-images.githubusercontent.com/96809909/155627771-165d2538-c320-4a49-8b73-ad5de2ae6d15.gif)